src/Discord.Net.Commands/Builders/ModuleClassBuilder.cs

```diff
@@ -249,7 +249,7 @@ private static bool IsValidModuleDefinition(TypeInfo typeInfo)
         private static bool IsValidCommandDefinition(MethodInfo methodInfo)
         {
             return methodInfo.IsDefined(typeof(CommandAttribute)) &&
-                   methodInfo.ReturnType == typeof(Task) &&
+                   (methodInfo.ReturnType == typeof(Task) || methodInfo.ReturnType == typeof(Task<RuntimeResult>)) &&
```

Comment:
Since this PR (allegedly) still needs a discussion, I'm throwing this note in here: this check won't work *entirely* well when people want to return a `Task<CustomRuntimeResult>`.